gsk4/Gir.toml

```diff
@@ -99,12 +99,6 @@ status = "generate"
     [[object.function]]
     name = "new"
     manual = true
-    [[object.function]]
```

Comment:
please remove this commit, those functions are not generated automatically and as shown below, they cannot be yet.

Reply:
This is automatically detected by `Gir` now, so the annotation is no longer necessary. We discussed it there: https://github.com/fengalin/gtk4-rs/commit/4bf3298cacb6654b6834c50153c4095d72a2648f#r157572515, or did I miss something?

Reply:
They should not be generated as commented code, that has always been the case...

Reply:
and without it we don't generate docs

Reply:
ok removed